tests/test_atomic/test_request.py

```diff
@@ -80,8 +80,8 @@ class TestAtomicOperationRequest:
         ],
     )
     def test_request_data(self, operation_request: dict):
-        validated = AtomicOperationRequest.parse_obj(operation_request)
-        assert validated.dict(exclude_unset=True, by_alias=True) == operation_request
+        validated = AtomicOperationRequest.model_validate(operation_request)
+        assert validated.model_dump(exclude_unset=True, by_alias=True) == operation_request
 
     def test_not_supported_operation(
```

Comment:
ValidationError now returns a new error message but I'm uncertain if we should modify the test itself to accommodate this change. I couldn't find where pydantic sets this new error message, which makes me hesitant to update the test without understanding the reason behind the change